data-prepper-plugins/otel-trace-source/src/main/java/com/amazon/dataprepper/plugins/certificate/acm/ACMCertificateProvider.java

```diff
@@ -53,6 +54,9 @@ public ACMCertificateProvider(final AWSCertificateManager awsCertificateManager,
                                   final String passphrase) {
         this.awsCertificateManager = Objects.requireNonNull(awsCertificateManager);
         this.acmArn = Objects.requireNonNull(acmArn);
+        if(!acmArn.equals(Arn.fromString(acmArn).toString())) {
```

Comment:
Same as above

Reply:
I think all we need is to call `Arn.fromString(acmArn)`.
You can see in [their implementation](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/arn/Arn.java#L124-L157) that they throw `IllegalArgumentException` when it is not an ARN.

Reply:
Yes, I've updated the code to throw the custom exception.